code/modules/research/techweb/all_nodes.dm

```diff
@@ -491,10 +491,8 @@
 	id = "basic_tools"

```

Comment:
just noticed, move this to the top of the file with the rest of the starting nodes, then all good to merge.